src/prefect/cli/deploy.py

```diff
@@ -194,6 +194,14 @@ async def deploy(
             " provided."
         ),
     ),
+    build_push_once: bool = typer.Option(
+        False,
+        "--build-push-once",
+        help=(
```

Comment:
I would appreciate feedback on whether it should be a single flag like `--build-push-once`, `--build once` or something else :)